### DISCUSSION TITLE
Queue worker not found error in app log when using SQLite database

### DISCUSSION THREAD

**Discussion by aannara** [fastapi/fastapi#8461] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from pathlib import Path
from typing import List

from fastapi import Body, Depends, FastAPI, HTTPException, Response
from fastapi.responses import FileResponse, HTMLResponse, JSONResponse, RedirectResponse
from sqlalchemy.orm import Session

from . import config, crud, models, schemas
from .database import SessionLocal, engine

models.Base.metadata.create_all(bind=engine)

app = FastAPI(title="Foo", description="Bar")

# Dependency
def get_db():
    db = SessionLocal()
    try:
        yield db
    finally:
        db.close()

@app.get("/favicon.ico", include_in_schema=False)
async def favicon():
    return FileResponse(config.FAVICON_PATH)

@app.get("/items", response_model=List[schemas.MyModel], tags=["items"])
def get_all_items(skip: int = 0, limit: int = 100, db: Session = Depends(get_db)):
    items = crud.get_all_items(db, skip=skip, limit=limit)
    return items
```

### Description

I am following the excellent documentation of FastAPI itself to create this app (https://fastapi.tiangolo.com/tutorial/sql-databases/)
I have a simple FastAPI which is deployed in my own server.
The app connects to a SQLite database which resides in the server itself.
After I start the application, in the app log, I continuously get the below message and I do not know how to stop it.

```
INFO:     Uvicorn running on http://0.0.0.0:8001 (Press CTRL+C to quit)
INFO:     Started reloader process [12097] using watchgod
INFO:     Started server process [12099]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     <ip>:0 - "GET /docs HTTP/1.1" 200 OK
INFO:     <ip>:0 - "GET /openapi.json HTTP/1.1" 200 OK
INFO:     <ip>:0 - "GET /api/v1/items?skip=0&limit=100 HTTP/1.1" 200 OK
INFO:     <ip>:0 - "GET /queue/worker HTTP/1.1" 404 Not Found
INFO:     <ip>:0 - "GET /queue/worker HTTP/1.1" 404 Not Found
INFO:     <ip>:0 - "GET /queue/worker HTTP/1.1" 404 Not Found
```

I suspect this is because I am connecting to SQLite DB which does not support async operations natively and the methods in my main.py start with **def foo** instead of **async def foo**

How do I get rid of this message? **"GET /queue/worker HTTP/1.1" 404 Not Found**
It is polluting my logs currently. Can I fix it or have a way to suppress it?

### Operating System

Linux

### Operating System Details

SLES12 SP5

### FastAPI Version

0.75.0

### Python Version

Python 3.7.4

### Additional Context

I am following the excellent documentation of FastAPI itself to create this app (https://fastapi.tiangolo.com/tutorial/sql-databases/)
And I am connecting to a SQLite DB.

**Comment by jonatasoli** [comment]
Is possible you share your database.py file?

**Comment by aannara** [comment]
`database.py`
```
from sqlalchemy import create_engine
from sqlalchemy.ext.declarative import declarative_base
from sqlalchemy.orm import sessionmaker

from . import config

SQLALCHEMY_DATABASE_URL = f"sqlite:///./{config.DATABASE_FOLDER}/{config.APP_NAME}.db"

engine = create_engine(
    SQLALCHEMY_DATABASE_URL, connect_args={"check_same_thread": False}
)
SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)

Base = declarative_base()
```

@jonatasoli Do you think I need to change something in my database connection mechanism?

**Comment by jonatasoli** [comment]
@aannara I'm trying to reproduce your error but I'm not able to, can you tell me the version of python you are using and the version of fastapi and sqlalchemy?
If possible, can you also give me a gist of a more simplified version of your case to facilitate the reproduction of the error?

**Comment by aannara** [comment, marked as answer]
Hi all,

I should have replied long ago. I was able to root cause the problem. I was using nginx as a reverse proxy to this URL, which is why I got the unexpected `"GET /queue/worker HTTP/1.1" 404 Not Found` . I ended not using nginx ultimately so I do not know how to fix this, but if I turn off the nginx service, these messages are not shown anymore
